asciipic/web/js/prepareJsonForServer.js

```
@@ -1,112 +1,128 @@
-                pictureSizeHeight: pictureSizeHeightInput,
-                pictureSizeWidth: pictureSizeWidthInput,
-                userAgent: userAgentInput,
-                filtersType: filtersTypeInput,
-                causes: causesInput,
-                time:timeInput,
-                typeOfTime: typeOfTimeInput
-            };
-    //verif
-    document.getElementById('statisticResults').innerHTML = informationsJSON['mainCommand'] 
-                                                            + "<br>" + informationsJSON['ip'] 
-                                                            + "<br>" + informationsJSON['tag'] 
-                                                            + "<br>" + informationsJSON['postDate'] 
-                                                            + "<br>" + informationsJSON['pictureSizeHeight'] 
-                                                            + "<br>" + informationsJSON['pictureSizeWidth'] 
-                                                            + "<br>" + informationsJSON['userAgent']
-                                                            + "<br>" + informationsJSON['filtersType']
-                                                            + "<br>" + informationsJSON['causes']
-                                                            + "<br>" + informationsJSON['time']
-                                                            + "<br>" + informationsJSON['typeOfTime'];
+        /* NOTE(AlexandraFolvaiter): fix it!*/
+        //send request to server
+         e.preventDefault();
+        $.ajax({
+        url: "http://localhost:8084/v1/statistic",
+        headers: {
+            "content-type": "application/json"
+          },
+        type: "POST",
+        data: informationsJSON,
+        dataType: "json",
+        "processData": false,
+        success: function(response) {
+          alert(response);
+        },
+        error: function(response) {
+          console.log(response);
```

This ignores the error, a warning message should appear ( maybe a pop up .. )